gnd/src/test/java/com/google/android/gnd/persistence/local/room/converter/ResponseJsonConverterTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.local.room.converter;
+
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.model.form.Field.Type;
+import com.google.android.gnd.model.observation.DateResponse;
+import com.google.android.gnd.model.observation.Response;
+import com.google.android.gnd.model.observation.TimeResponse;
+import com.google.android.gnd.persistence.local.LocalDatabaseModule;
+import com.google.android.gnd.rx.SchedulersModule;
+import dagger.hilt.android.testing.HiltAndroidTest;
+import dagger.hilt.android.testing.HiltTestApplication;
+import dagger.hilt.android.testing.UninstallModules;
+import java.util.Date;
+import java8.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@HiltAndroidTest
+@UninstallModules({SchedulersModule.class, LocalDatabaseModule.class})
+@Config(application = HiltTestApplication.class)
+@RunWith(RobolectricTestRunner.class)
+public class ResponseJsonConverterTest {
+
+  //Date represented in YYYY-MM-DDTHH:mmZ Format from 1632501600000L milliseconds.
```

Comment:
Nit: Missing space after "//".

Please use google-java-format to format all code. You can trigger this automatically on save in Android Studio if you prefer.

Reply:
updated.